functions.js

```diff
@@ -0,0 +1,214 @@
+//1a==========================
+const allTvNames = inventory.map((item) => {
+    return item.name
+})
+console.log(allTvNames)
+
+
+//1b==========================
+const soldOutTvs = inventory.filter((item) => {
+    return item.sold === item.originalStock
+})
+console.log(soldOutTvs)
+
+
+//1c==========================
+const ambiLightTvs = inventory.filter((item) => {
+    return item.options.ambiLight
+})
+console.log(ambiLightTvs)
+
+
+//1d==========================
+const priceSort = inventory.sort((a, b) => {
+    return a.price - b.price
+})
+console.log(priceSort)
+
+
+//2a========================================================================================================
+const marketing = document.getElementById('marketing')
+
+function amountOfTvsSold(list) {
+    let amount = 0
+    list.map((item) => {
+        amount += item.sold
+    })
+
+    return amount
+}
+
+console.log(amountOfTvsSold(inventory))
+
+
+//2b
+const html_sold = document.createElement('p')
+html_sold.setAttribute('class', 'sold')
+html_sold.textContent = amountOfTvsSold(inventory).toString() + " tv's sold"
+marketing.appendChild(html_sold)
+
+
+//2c==========================
+function amountOfTvsBought(list) {
+    let amount = 0
+    for (let item of list) {
+        amount += item.originalStock
+    }
+    return amount
+}
+
+console.log(amountOfTvsBought(inventory))
+
+
+//2d==========================
+const html_bought = document.createElement('p')
+html_bought.setAttribute('class', 'bought')
+html_bought.textContent = amountOfTvsBought(inventory) + " tv's bought"
+marketing.appendChild(html_bought)
+
+
+//2e==========================
+const html_inStock = document.createElement('p')
+html_inStock.setAttribute('class', 'stock')
+html_inStock.textContent = (amountOfTvsBought(inventory) - amountOfTvsSold(inventory)).toString() + " in stock"
+marketing.appendChild(html_inStock)
+
+
+//3a========================================================================================================
+const allBrands = document.getElementById('allBrands')
+
+inventory.map((item) => {
+    let html_allBrands = document.createElement('p')
+    html_allBrands.textContent = item.brand
+    allBrands.appendChild(html_allBrands)
+})
+
+
+//3b==========================
+function getAllTvBrands(list) {
+    return list.map((item) => {
+        return item.brand
+    })
+}
+
+console.log(getAllTvBrands(inventory))
+
+
+//4a========================================================================================================
+function tvName(tv) {
+    return tv.brand + " " + tv.type + " - " + tv.name
```

Comment:
Wellicht iets beter te lezen op deze manier
```suggestion
    return `${tv.brand} ${tv.type} - ${tv.name}`;
```